os/arch/arm/src/armv8-m/Toolchain.defs

```diff
@@ -224,9 +225,9 @@ ifeq ($(CONFIG_ARMV8M_TOOLCHAIN),CODEREDW)
   MAXOPTIMIZATION ?= -Os
   ifeq ($(CONFIG_ARCH_CORTEXM33),y)
     ifeq ($(CONFIG_ARCH_FPU),y)
-      ARCHCPUFLAGS = -mcpu=cortex-m33 -mthumb -mfpu=fpv4-sp-d16 -mfloat-abi=hard
+      ARCHCPUFLAGS = -mcpu=cortex-m33 -mthumb -mcmse -mfpu=fpv4-sp-d16 -mfloat-abi=hard
```

Comment:
do we need to change to fpv5 for windows also?

Reply:
we are just adding the cmse flag here

Reply:
this is related to floating-point h/w available on target. how it varies for different build platforms?

Reply:
This is fixed in latest commit. You are right the fpu needs to be common, I got confused betweeen the fpu flag and cmse